[CI] XPackRestIT "transform/transforms_stats/Test get continuous transform stats" failure

This failure was part of a longer list of failed tasks in XPackRestIT, but this one seems to have been the first failing one and looks legit. For reference, I've also created https://github.com/elastic/elasticsearch/issues/52428. Also, it doesn't reproduce for me.

```
08:01:28   1> [2020-02-16T23:43:11,666][INFO ][o.e.x.t.r.XPackRestIT    ] [test] [p0=transform/transforms_stats/Test get continuous transform stats] before test
08:01:28   1> [2020-02-16T23:43:13,044][INFO ][o.e.x.t.r.XPackRestIT    ] [test] Stash dump on test failure [{
08:01:28   1>   "stash" : {
08:01:28   1>     "body" : {
08:01:28   1>       "count" : 1,
08:01:28   1>       "transforms" : [
08:01:28   1>         {
08:01:28   1>           "id" : "airline-transform-stats-continuous",
08:01:28   1>           "state" : "started",
08:01:28   1>           "node" : {
08:01:28   1>             "id" : "SMnPdQK0T-SFKbe2hgOQ6Q",
08:01:28   1>             "name" : "integTest-0",
08:01:28   1>             "ephemeral_id" : "DhvyWYLlR8qwMGK2ANFG1w",
08:01:28   1>             "transport_address" : "127.0.0.1:45379",
08:01:28   1>             "attributes" : { }
08:01:28   1>           },
08:01:28   1>           "stats" : {
08:01:28   1>             "pages_processed" : 1,
08:01:28   1>             "documents_processed" : 0,
08:01:28   1>             "documents_indexed" : 0,
08:01:28   1>             "trigger_count" : 1,
08:01:28   1>             "index_time_in_ms" : 0,
08:01:28   1>             "index_total" : 0,
08:01:28   1>             "index_failures" : 0,
08:01:28   1>             "search_time_in_ms" : 2,
08:01:28   1>             "search_total" : 1,
08:01:28   1>             "search_failures" : 0,
08:01:28   1>             "exponential_avg_checkpoint_duration_ms" : 47.0,
08:01:28   1>             "exponential_avg_documents_indexed" : 0.0,
08:01:28   1>             "exponential_avg_documents_processed" : 0.0
08:01:28   1>           },
08:01:28   1>           "checkpointing" : {
08:01:28   1>             "last" : {
08:01:28   1>               "checkpoint" : 1,
08:01:28   1>               "timestamp_millis" : 1581914592903,
08:01:28   1>               "time_upper_bound_millis" : 1581914532903
08:01:28   1>             }
08:01:28   1>           }
08:01:28   1>         }
08:01:28   1>       ]
08:01:28   1>     }
08:01:28   1>   }
08:01:28   1> }]
08:01:28   1> [2020-02-16T23:43:23,577][INFO ][o.e.x.t.r.XPackRestIT    ] [test] There are still tasks running after this test that might break subsequent tests [data_frame/transforms[c]].
08:01:28   1> [2020-02-16T23:43:23,578][INFO ][o.e.x.t.r.XPackRestIT    ] [test] [p0=transform/transforms_stats/Test get continuous transform stats] after test
08:01:28   2> REPRODUCE WITH: ./gradlew ':x-pack:plugin:integTestRunner' --tests "org.elasticsearch.xpack.test.rest.XPackRestIT.test {p0=transform/transforms_stats/Test get continuous transform stats}" -Dtests.seed=FF7502DFD4FB6063 -Dtests.security.manager=true -Dtests.locale=mt -Dtests.timezone=US/Eastern -Dcompiler.java=13 -Dtests.rest.blacklist=getting_started/10_monitor_cluster_health/*
08:01:28   2> REPRODUCE WITH: ./gradlew ':x-pack:plugin:integTestRunner' --tests "org.elasticsearch.xpack.test.rest.XPackRestIT.test {p0=transform/transforms_stats/Test get continuous transform stats}" -Dtests.seed=FF7502DFD4FB6063 -Dtests.security.manager=true -Dtests.locale=mt -Dtests.timezone=US/Eastern -Dcompiler.java=13 -Dtests.rest.blacklist=getting_started/10_monitor_cluster_health/*
08:01:28   2> java.lang.AssertionError: Failure at [transform/transforms_stats:250]: transforms.0.stats.exponential_avg_checkpoint_duration_ms didn't match expected value:
08:01:28     transforms.0.stats.exponential_avg_checkpoint_duration_ms: expected Double [0.0] but was Double [47.0]
08:01:28         at org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase.executeSection(ESClientYamlSuiteTestCase.java:405)
08:01:28         at org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase.test(ESClientYamlSuiteTestCase.java:382)
08:01:28         at jdk.internal.reflect.GeneratedMethodAccessor12.invoke(Unknown Source)
08:01:28         at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
08:01:28         at java.base/java.lang.reflect.Method.invoke(Method.java:566)
08:01:28         at com.carrotsearch.randomizedtesting.RandomizedRunner.invoke(RandomizedRunner.java:1754)
08:01:28         at com.carrotsearch.randomizedtesting.RandomizedRunner$8.evaluate(RandomizedRunner.java:942)
08:01:28         at com.carrotsearch.randomizedtesting.RandomizedRunner$9.evaluate(RandomizedRunner.java:978)
08:01:28         at com.carrotsearch.randomizedtesting.RandomizedRunner$10.evaluate(RandomizedRunner.java:992)
08:01:28         at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
08:01:28         at org.apache.lucene.util.TestRuleSetupTeardownChained$1.evaluate(TestRuleSetupTeardownChained.java:49)
08:01:28         at org.apache.lucene.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:45)
08:01:28         at org.apache.lucene.util.TestRuleThreadAndTestName$1.evaluate(TestRuleThreadAndTestName.java:48)
08:01:28         at org.apache.lucene.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:64)
08:01:28         at org.apache.lucene.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:47)
08:01:28         at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
08:01:28         at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:370)
08:01:28         at com.carrotsearch.randomizedtesting.ThreadLeakControl.forkTimeoutingTask(ThreadLeakControl.java:819)
08:01:28         at com.carrotsearch.randomizedtesting.ThreadLeakControl$3.evaluate(ThreadLeakControl.java:470)
08:01:28         at com.carrotsearch.randomizedtesting.RandomizedRunner.runSingleTest(RandomizedRunner.java:951)
08:01:28         at com.carrotsearch.randomizedtesting.RandomizedRunner$5.evaluate(RandomizedRunner.java:836)
08:01:28         at com.carrotsearch.randomizedtesting.RandomizedRunner$6.evaluate(RandomizedRunner.java:887)
08:01:28         at com.carrotsearch.randomizedtesting.RandomizedRunner$7.evaluate(RandomizedRunner.java:898)
08:01:28         at org.apache.lucene.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:45)
08:01:28         at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
08:01:28         at org.apache.lucene.util.TestRuleStoreClassName$1.evaluate(TestRuleStoreClassName.java:41)
08:01:28         at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
08:01:28         at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
08:01:28         at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
08:01:28         at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
08:01:28         at org.apache.lucene.util.TestRuleAssertionsRequired$1.evaluate(TestRuleAssertionsRequired.java:53)
08:01:28         at org.apache.lucene.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:47)
08:01:28         at org.apache.lucene.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:64)
08:01:28         at org.apache.lucene.util.TestRuleIgnoreTestSuites$1.evaluate(TestRuleIgnoreTestSuites.java:54)
08:01:28         at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
08:01:28         at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:370)
08:01:28         at com.carrotsearch.randomizedtesting.ThreadLeakControl.lambda$forkTimeoutingTask$0(ThreadLeakControl.java:826)
08:01:28         at java.base/java.lang.Thread.run(Thread.java:834)
08:01:28 
08:01:28         Caused by:
08:01:28         java.lang.AssertionError: transforms.0.stats.exponential_avg_checkpoint_duration_ms didn't match expected value:
08:01:28         transforms.0.stats.exponential_avg_checkpoint_duration_ms: expected Double [0.0] but was Double [47.0]
08:01:28             at org.elasticsearch.test.rest.yaml.section.MatchAssertion.doAssert(MatchAssertion.java:93)
08:01:28             at org.elasticsearch.test.rest.yaml.section.Assertion.execute(Assertion.java:76)
08:01:28             at org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase.executeSection(ESClientYamlSuiteTestCase.java:398)
08:01:28             ... 37 more
```

https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+master+matrix-java-periodic/ES_RUNTIME_JAVA=java11,nodes=general-purpose/516/console
[7.6.1] https://gradle-enterprise.elastic.co/s/dcsdwsaodnlso
[7.7.0] https://gradle-enterprise.elastic.co/s/aeqrfacqyj24u
[7.6.1] https://gradle-enterprise.elastic.co/s/vrp66sw4c6zyy
[7.7.0] https://gradle-enterprise.elastic.co/s/zpd4kfpizyq7c
https://gradle-enterprise.elastic.co/s/frzrlcgeo45j4

--- a/x-pack/plugin/src/test/resources/rest-api-spec/test/transform/transforms_stats.yml
+++ b/x-pack/plugin/src/test/resources/rest-api-spec/test/transform/transforms_stats.yml
@@ -247,7 +247,7 @@ teardown:
   - gte: { transforms.0.stats.search_time_in_ms: 0 }
   - lte: { transforms.0.stats.search_total: 1 }
   - match: { transforms.0.stats.search_failures: 0 }
-  - match: { transforms.0.stats.exponential_avg_checkpoint_duration_ms: 0.0 }
+  - is_true: transforms.0.stats.exponential_avg_checkpoint_duration_ms
   - match: { transforms.0.stats.exponential_avg_documents_indexed: 0.0 }
   - match: { transforms.0.stats.exponential_avg_documents_processed: 0.0 }
 

